confignode/src/main/java/org/apache/iotdb/confignode/manager/load/LoadManager.java

```diff
@@ -185,20 +198,26 @@ private void updateLoadStatistics() {
     // Broadcast the RegionRouteMap if some LoadStatistics has changed
     boolean isNeedBroadcast = false;
 
-    // Update NodeStatistics
-    Map<Integer, NodeStatistics> differentNodeStatisticsMap = new ConcurrentHashMap<>();
+    // Update NodeStatistics:
+    // NodeStatistics[]:index 0 means the current NodeStatistics, index 1 means the previous
+    // NodeStatistics
+    Map<Integer, Pair<NodeStatistics, NodeStatistics>> differentNodeStatisticsMap =
+        new ConcurrentHashMap<>();
```

Comment:
Maybe encapsulate the node state difference as a Class is a better choice, and some common logic can be implemented inside such class in future work.